packages/strapi-parts/src/Popover/Popover.stories.mdx

```diff
@@ -56,9 +58,7 @@ Description...
   </Story>
 </Canvas>
 
-## Popover
-
-Description...
+## Open by default
```

Comment:
Not sure about this one..

Reply:
What do you mean by "open by default"?

Reply:
The popover seems to be open from the beginning
![Screenshot 2021-09-29 at 14 18 35](https://user-images.githubusercontent.com/4447713/135266935-35cc1f60-fdd3-4028-888b-26e027c9ebd5.png)

Reply:
oh okay, that shouldn't probably be the case. Usually an open popover is triggered by an action first